hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/TransferContext.java

```diff
@@ -117,6 +117,12 @@ public interface TransferContext {
      */
     boolean isEnforceMonoServiceRestrictionsOnAutoCreationCustomFeePayments();
 
+    /**
+     * Indicates if this transfer should override the paused token and frozen account checks.
+     * @return whether the transfer should override the checks
+     */
+    boolean shouldOverrideFreezeAndPauseStatusChecks();
+
```

Comment:
can be changed to `allowFreezeAndPausedTokenTransfer` ?

Reply:
Also the javadoc should indicate it is true only for `TokenReject`

Reply:
I agree its a better naming. Changed it to `allowFreezeAndPausedTokenTransfer`.

I've also adjusted the Javadoc. 🙂